great_expectations/core/expectation_diagnostics/expectation_diagnostics.py

```diff
@@ -330,6 +330,12 @@ def _check_input_validation(
                     "passed": False,
                 }
             )
+
+        if "validate_configuration" not in expectation_instance.__class__.__dict__:
+            passed = False
```

Comment:
**Can you also update `great_expectations/expectations/core/expect_column_values_to_be_in_set.py`** to change

```
        if not super().validate_configuration(configuration):
            return False
```

to

```
        super().validate_configuration(configuration)
```

It's the **only** core or contrib Expectation that has a `return False`